src/utils/utils.ts

```diff
@@ -122,3 +123,52 @@ export function isNode() {
     typeof process.versions.node !== 'undefined'
   );
 }
+
+/**
+ * Convert a base64 string to a Uint8Array for Node.js and browser environments.
+ * @returns A Uint8Array
+ */
+export function base64ToBytes(base64String: string) {
```

Comment:
minor: could you explicitly annotate these functions with return values as well?

While Typescript can infer them, I'd find the extra clarify helpful, and it ensures the return type won't accidentally change if we update the function body.